src/js/tech/html5.js

```diff
@@ -95,9 +95,11 @@ class Html5 extends Tech {
     let emulatedTt = this.textTracks();
 
     // remove native event listeners
-    tt.removeEventListener('change', this.handleTextTrackChange_);
-    tt.removeEventListener('addtrack', this.handleTextTrackAdd_);
-    tt.removeEventListener('removetrack', this.handleTextTrackRemove_);
+    if (tt && tt.removeEventListener) {
```

Comment:
Yeah, that's probably safe. Also I think it'd be better if we didn't abbreviate both textTrack and textTracks to `tt` in different places. That threw me off for a second.